cartoframes/utils/geom_utils.py

```diff
@@ -255,11 +255,13 @@ def to_geojson(geom, buffer_simplify=True):
             return json.dumps(shapely.geometry.mapping(geom), sort_keys=True)
 
 
-def check_crs(gdf):
-    current_crs = get_crs(gdf)
-    expected_crs = 'epsg:4326'
-    if current_crs is not None and current_crs != expected_crs:
-        raise ValueError('No valid geometry CRS "{}", it must be "{}".'.format(current_crs, expected_crs))
+def is_reprojection_needed(gdf):
+    crs = get_crs(gdf)
```

Comment:
Why not `get_crs(gdf) == 'epsg:4326'`?

Reply:
because the crs can be `None`. The tests raised this case

Reply:
Let's fix the tests then. I tests should be updated to match the new behavior

Reply:
I have checked the tests, and they are failing because of a different reason (they expect exceptions, and now since the reprojection is automatic no exceptions are raised)

Reply:
Yes, I noticed that so I refactored and removed some tests :+1:

Reply:
That's the problem of doing the reviews "_live_" (I prefer asking explicitly for new CR rounds :))

Reply:
OK